packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1890,6 +1941,14 @@ export class ContainerRuntime
 				case ContainerMessageType.BlobAttach:
 					this.blobManager.processBlobAttachOp(message, local);
 					break;
+				case ContainerMessageType.IdAllocation:
+					if (this.runtimeOptions.enableRuntimeIdCompressor !== true) {
+						throw new Error(
```

Comment:
This kind of answers my question here - https://github.com/microsoft/FluidFramework/pull/14340/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#r1152549003.